src/category_theory/instances/Top/limits.lean

```diff
@@ -0,0 +1,63 @@
+-- Copyright (c) 2017 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Patrick Massot, Scott Morrison, Mario Carneiro
+
+import category_theory.instances.Top.basic
+import category_theory.limits.types
+import category_theory.limits.preserves
+
+open category_theory
+open category_theory.instances
+open topological_space
+
+universe u
+
+namespace category_theory.instances.Top
+
+open category_theory.limits
+
+variables {J : Type u} [small_category J]
+
+def limit (F : J ⥤ Top.{u}) : cone F :=
+{ X := ⟨limit (F ⋙ forget), ⨆ j, (F.obj j).str.induced (limit.π (F ⋙ forget) j)⟩,
+  π :=
+  { app := λ j, ⟨limit.π (F ⋙ forget) j, continuous_iff_induced_le.mpr (lattice.le_supr _ j)⟩,
+    naturality' := λ j j' f, subtype.eq ((limit.cone (F ⋙ forget)).π.naturality f) } }
+
+def limit_is_limit (F : J ⥤ Top.{u}) : is_limit (limit F) :=
+by refine is_limit.of_faithful forget (limit.is_limit _) (λ s, ⟨_, _⟩) (λ s, rfl);
```

Comment:
I agree this could be changed, but this isn't the relevant PR. :-) That line is already in mathlib, and you only see a change here because this PR is stacked on top of #920, which is solely rearranging files, but hasn't been merged yet.

Reply:
Ok, I see. Would you please change it over there?